docs/modules/ROOT/workflows/southbound_prog_workflow_v1.puml

```diff
@@ -0,0 +1,175 @@
+'        See the License for the specific language governing permissions and
+'        limitations under the License.
+
+@startuml
+
+title Southbound Workflow of Port Provisioning - v1
+
+skinparam monochrome true
+collections "Port Manager" as pm
+collections "Data Plane Manager" as dpm
+'collections "Message Queue" as mq
+participant "ACA@Host 1" as host1
+participant "ACA@Host 2" as host2
+participant "ACA@Host 3" as host3
+
+==Scenario A: Create the first Port P1 at Host 1==
+autonumber "<b>[0]"
+pm -> dpm: POST /port async with RequestBody NetworkConfiguration={\n\t[P1 w/ neighbor_info=NULL],[Subnet],[VPC],[SG]}
+activate dpm
+dpm -> host1: Send a GoalState message {[(P1,CREATE)],[(Subnet,INFO)]} via gPRC
+activate host1
+host1 -> dpm: Reply a GoalStateOperationReply message on P1 programming status
+deactivate host1
+dpm -> pm: Reply an InternalProgrammingStatus message on P1 programming status
+deactivate dpm
+
+==Create the second Port P2 at Host 1==
+autonumber 5 "<b>[0]"
+pm -> dpm: POST /port async with RequestBody NetworkConfiguration={\n\t[P2 w/ neighbor_info=NULL],[Subnet],[VPC],[SG]}
+activate dpm
+
+dpm -> host1: Send a GoalState message \n{[(P2,CREATE)],[(Subnet,INFO)]} via gPRC
+activate host1
+host1 -> dpm: Reply a GoalStateOperationReply message on P2 programming status
+deactivate host1
+
+dpm -> pm: Once receiving response from Host1, reply an InternalProgrammingStatus\nmessage on P2 programming status
+deactivate dpm
+==================================================
+
+==Scenario B: Create Ports (P1, P2) at Host 1==
+autonumber "<b>[0]"
+pm -> dpm: POST /port async with RequestBody NetworkConfiguration={\n\t[P1,P2 w/ neighbor_info=NULL],[Subnet],[VPC],[SG]}
+activate dpm
+
+autonumber 2 "<b>[0]"
+dpm -> host1: Send a GoalState message \n{[(P1,CREATE),(P2,CREATE)],[(Subnet,INFO)]} via gPRC
+activate host1
+host1 -> dpm: Reply a GoalStateOperationReply message on P1 and P2 programming status
+deactivate host1
+
+autonumber 4 "<b>[0]"
+dpm -> pm: After receiving response from Host1 and Host2, reply an InternalProgrammingStatus\nmessage on P2 programming status
+deactivate dpm
+
+
+==Scenario C: Create the first Port P1 at Host 1==
+autonumber "<b>[0]"
+pm -> dpm: POST /port async with RequestBody NetworkConfiguration={\n\t[P1 w/ neighbor_info=NULL],[Subnet],[VPC],[SG]}
+activate dpm
+dpm -> host1: Send a GoalState message {[(P1,CREATE)],[(Subnet,INFO)]} via gPRC
+activate host1
+host1 -> dpm: Reply a GoalStateOperationReply message on P1 programming status
+deactivate host1
+dpm -> pm: Reply an InternalProgrammingStatus message on P1 programming status
+deactivate dpm
+
+==Create the second Port P2 at Host 2==
+autonumber 5 "<b>[0]"
+pm -> dpm: POST /port async with RequestBody NetworkConfiguration={\n\t[P2 w/ neighbor_info=[P1]],[Subnet],[VPC],[SG]}
+activate dpm
+
+autonumber 6 "<b>[0.a]"
+dpm -> host2: Send a GoalState message \n{[(P2,CREATE),(P1,NEIGHBOR_CREATE_UPDATE)],[(Subnet,INFO)]} via gPRC
```

Comment:
Let's plan to change NEIGHBOR_CREATE_UPDATE to our new Neighbor state format: NEIGHBOR,CREATE(operation_type),L2(neighbor_type)

Reply:
Acked. Add @haboy52581 to update the workflow based on the latest schema.